src/current/v24.3/sql-feature-support.md

```diff
@@ -1,18 +1,16 @@
 ---
-title: SQL Feature Support in CockroachDB v24.2
+title: SQL Feature Support in CockroachDB
```

Comment:
Shouldn't it be `title: SQL Feature Support in CockroachDB v24.3` because version is mentioned in all the other version pages?

Reply:
It's harder to maintain because the version is in the title, so I took it out. The version number is now in the intro. I can update the remaining supported versions (v23.1 and later) if you think it's necessary. cc @shannonbradshaw

Reply:
Per our Slack discussion, I'm happy to have this change eliminate the version. https://cockroachlabs.slack.com/archives/G01Q35D80EM/p1734461664934839
I don't think it's strictly necessary that we update the titles for other versions, rather I'd prefer we eliminate the hard-coded version from the title going forward.